lib/v8.js

```diff
@@ -212,7 +213,8 @@ function getHeapCodeStatistics() {
   return {
     code_and_metadata_size: buffer[kCodeAndMetadataSizeIndex],
     bytecode_and_metadata_size: buffer[kBytecodeAndMetadataSizeIndex],
-    external_script_source_size: buffer[kExternalScriptSourceSizeIndex]
+    external_script_source_size: buffer[kExternalScriptSourceSizeIndex],
+    cpu_profiler_metadata_size: buffer[kCPUProfilerMetaDataSizeIndex]
```

Comment:
Could you please add
```
* cpu_profiler_metadata_size: number;
```
to the JSDoc few lines above in https://github.com/nodejs/node/blob/daae938f32f2660a8cb7bf42891bc74b984c7f36/lib/v8.js#L208-L212?

Reply:
done, thanks.